### DISCUSSION TITLE
Approved flag location/exposure

### DISCUSSION THREAD

**Discussion by glwitzenburg** [babel/babel#14922] [Q&A] [answered]
- We are doing an audit to check the usage of our translations inside of our application, and that part has been completed. Now we are trying take our values that have been approved and check that box for each respective k/v translation. Is there a way that this T/F values for approved is exposed with a csv/xlsx file? When I download the csv/xslx file I see all of the IDs and respective translations but would like to know that flag so I can programmatically update this, rather than having to do it by hand.

**Comment by nicolo-ribaudo** [comment, marked as answer]
Hi! There are multiple tools called "Babel", you probably asked this in the wrong place :)
